src/components/ReplayForm.vue

```diff
@@ -237,26 +250,26 @@ Civ draft: ${extractDraftUrl(civDraft.value)}`
         if (newMeta?.maps?.pickedMaps?.length) {
           const numOfMaps = newMeta.maps.pickedMaps.length
           if (numOfMaps % 2 == 0) {
-            gamesStore.setGamesNumber(numOfMaps - 1)
+            setExpectedGamesCount(numOfMaps + 1)
           } else {
-            gamesStore.setGamesNumber(numOfMaps)
+            setExpectedGamesCount(numOfMaps)
           }
         }
         meta = newMeta
         metaErrors = newErrors
       }
     "
   />
-  <SetInfo
-    :games-count="gamesStore.games.length"
-    @set-games="gamesStore.setGamesNumber"
-    @set-bo-pa="(newBoPa) => (boPa = newBoPa)"
-  />
```

Comment:
I moved the `SetInfo` closer to the download button since I expect people to forget to set that. Best to have it closer to the point where they see the error message.

Reply:
The idea is fine but I really don't like that the box effectively jumps around when you click since it affects the vertical size of the section above. We can revisit this later maybe with some auto-scrolling or other solution but I'm moving it back for now.